action/protocol/poll/staking_committee.go

```diff
@@ -247,22 +251,27 @@ func (sc *stakingCommittee) CalculateUnproductiveDelegates(
 }
 
 func (sc *stakingCommittee) Delegates(ctx context.Context, sr protocol.StateReader) (state.CandidateList, error) {
+	ctx = protocol.WithFeatureWithHeightCtx(ctx)
```

Comment:
is it possible to find a common place to add `WithFeatureWithHeightCtx`,
instead of doing it for every API?